src/python/Utils/CPMetrics.py

```diff
@@ -237,6 +237,8 @@ def flattenStats(cpdata):
     "Flatten cherrypy stats to make them suitable for monitoring"
     if isinstance(cpdata, str):
         cpdata = json.loads(cpdata)
```

Comment:
To load this json you need to have unicode data. I would suggest to use this utilitarian method instead:
https://github.com/dmwm/WMCore/blob/master/src/python/Utils/Utilities.py#L203

Can you please replace this if/else by the following line:
```
cpdata = json.loads(decodeBytesToUnicode(cpdata))
```
?

Reply:
done